src/views/QuantumTP6/index.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { withStyles } from '@material-ui/core';
 import PropTypes from 'prop-types';
 import Grid from '@material-ui/core/Grid';
-import { frum } from '../utils/query_ops';
-import { TP6_PAGES } from './config';
-import DashboardPage from '../components/DashboardPage/index';
-import PerfherderGraphContainer from '../containers/PerfherderGraphContainer/index';
+import frum from '../../utils/queryOps';
+import { TP6_PAGES } from '../../quantum/config';
```

Comment:
Eventually I would like the config to move directly under `srv/views`. It's fine for now.